### ISSUE TITLE
Schema parameter

### ISSUE THREAD

**Issue by fehrenmann** [dmarkey/pgdatadiff#3]
Hi David

Good job, i was searching for a light tool to compare two databases, and pgdatadiff largely meets my expectations. 

This is not an issue, but i have a question about schemas. Your package, for the connection, uses only the public schema, right ? What about a parameter --schema other_schema_than_public ? I think it would be very helpful..it wouldn't check that the schemas are the same, but allow to compare different schemas...

Take care and thanks a lot !

François

**Comment by dmarkey** [comment]
Yes good feature request.. PRs welcome or I can do it when I'm bored.
